src/components/questions/RecordUpload.tsx

```diff
@@ -0,0 +1,158 @@
+import AudioRecorderPlayer, {
+  AVEncoderAudioQualityIOSType,
+  AVEncodingOption,
+  AudioEncoderAndroidType,
+  AudioSet,
+  AudioSourceAndroidType,
+  RecordBackType,
+  PlayBackType,
+} from 'react-native-audio-recorder-player';
+import { Alert, PermissionsAndroid, Platform, StyleSheet, View } from 'react-native';
+import React, { Component } from 'react';
+
+import Icon from 'react-native-vector-icons/FontAwesome';
+import COLORS from '../../assets/colors';
+
+interface State {
+  isPlaying: boolean;
+  isRecording: boolean;
+  onVoiceSelected: string;
+}
+class RecordUpload extends Component<any, State> {
+  private audioRecorderPlayer: AudioRecorderPlayer;
+
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      isRecording: false,
+      isPlaying: false,
+      onVoiceSelected: '',
+    };
+
+    this.audioRecorderPlayer = new AudioRecorderPlayer();
+    this.audioRecorderPlayer.setSubscriptionDuration(0.1);
+  }
+
+  public render() {
+    return (
```

Comment:
Ui vindt ik op dit moment erg onduidelijk. Je kan niet zien of er al een opname is. Stel ik ben aan het opnemen en ik druk op play, wat gebeurd er dan? Hoe lang is mijn opname? Als er geen opname is kan ik op het terugspoel knopje drukken en op de prullenmand zonder melding.
Zou graag iets zien zoals in deze preview: https://www.npmjs.com/package/react-native-audio-recorder-player

Reply:
Wel slimme feedback had het wel afgedekt met alert op alleen play button, trash en replay iocon doen dan nog niks, voor je eerste comment. Ik heb hiervoor aantal suggesties. Voor voice recording wil je hiervoor een apart scherm in het algeheel? Want khad dit ongeveer als beeld. Voor beter overzicht, over de vraag. Kan namelijk zijn dat er meerdere antwoordopties kunnen zitten aan een vraag.
![image](https://user-images.githubusercontent.com/45880258/167252220-fe310d08-45cf-4417-bf87-2c355097fc9b.png)
 ik zou wel naast de button de afspeeltijd met afspelende tijd erbij kunnen zetten zoals dit 
![image](https://user-images.githubusercontent.com/45880258/167252289-1358be1a-477a-4511-aca5-86cf160b2450.png)
ik kan er ook voor zorrgen dat visibility van play, replay en trash icon invisible zet totdat de persoon op de mic icon selecteert. Ook wanneer recording dan word weggegooid zou ik ze dan weer invisible kunnen zetten.

Reply:
Alleen die balk met tijden erbij lijkt me voldoende, dan weet je als gebruiker ook hoelang de opaname is en hoever je bent met replay

Reply:
Ik zou de knoppen eerder disabled maken dan volledig weghalen, lijkt me er wat netter uitzien